workspaces/liferay-sample-workspace/configs/local/portal-ext.properties

```diff
@@ -6,6 +6,7 @@
 #jdbc.default.username=root
 #jdbc.default.password=
 
+feature.flag.LPD-27222=true
```

Comment:
Looks like these should be sorted.

Reply:
@ethib137 I thought so too, but I checked portal.properties, and the sorting there matches this because tickets with fewer digits come before tickets with more digits. Here, 5 digits precedes 6.

Reply:
That being said, Brian may just do a "Sublime Text" sort on this, and I'm not sure which wins in that case (not at my machine to check).

Reply:
oh haha... good to know. On my machine Sublime keeps the current sort as well, so hopefully that's correct 😂

Reply:
@drewbrokke I added the feature flag to the portal-ext.properties file and deployed the language folder. The feature flag wasn't enable... how does it work? Are there docs for this?